### ISSUE TITLE
Different result from by_slugs query and absolute URL query

### ISSUE THREAD

**Issue by Akswii** [storyblok/storyblok-docs#89]
Not sure if this is the right place to post this, but I've noticed that the Storyblok API handles slugs differently depending on which method you choose to fetch with.

If I fetch one story with `https://api.storyblok.com/v1/cdn/stories/posts/my-third-post/` then it doesn't matter if I use a trailing slash ( / ) or not - it will always fetch the story for me. But if I try to do the same single story query with the multiple story parameter (by_slugs) `https://api.storyblok.com/v1/cdn/stories/?by_slugs=posts/my-third-post` then the trailing slash would not match with an entry, only folders and their root entry.

This strikes me as weird as it isn't possible to create both a folder and a entry with the same slug. With this restriction in place I would expect both of these methods to function the same way and return the same stories.

In summary: 

When i query in the URL:
/posts/my-third-post <- works for an entry and a folder with a root entry
/posts/my-third-post/ <- works for an entry and a folder with a root entry

When i query by slugs:
`by_slugs=posts/my-third-post` <- works for an entry but not a folder with a root entry
`by_slugs=posts/my-third-post/` <- works for a folder with a root entry but not an entry

**Comment by waldbewohner** [comment]
you find a solution?
